dace/codegen/targets/cpu.py

```diff
@@ -228,7 +228,9 @@ def allocate_array(self, sdfg, dfg, state_id, node, function_stream,
 
         # Compute array size
         arrsize = nodedesc.total_size
-        arrsize_bytes = arrsize * nodedesc.dtype.bytes
+        arrsize_bytes = 42
```

Comment:
The value `42` has no special meaning here (other than the well-known one). Furthermore, it is not possible to get the size for an opaque type (by definition of it being opaque). However, this is fine since the value is not used when generating code with opaque types. The issue was that at some point, the code generator was trying to access `arrsize_bytes`, resulting in an error. Thus the need for initializing `arrsize_bytes` with some value. This was apparently rectified in another PR, so I removed the (apparently) obsolete initialization line.